CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Added a reference to Blank Starter Project in the README.
 - Added Frames Per Second (FPS) and Unity heap usage as metrics sent by `MetricSendSystem.cs`.
 - Added a warning message to the top of schema files copied into the `from_gdk_packages` directory.
+- Added an `ISnapshottable<T>` interface to all generated components. This allows you to convert a component to a snapshot.
```

Comment:
Can/should `T` be be something other than `Snapshot`?

Reply:
No

Reply:
It can't be used without. What type would be returned if it's not generic?

Reply:
Discussed offline, I confused the `Snapshot` type with `Improbable.Gdk.Core.Snapshot`